### PR TITLE
Obmin 267 back button

### PR DESCRIPTION
**Pull request by Voloshkina**
Created back button

### CODE REVIEW

File: src/components/back-button/BackButton.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export const Default = Template.bind({});
+Default.args = {
+  text: 'Button',
+};
+
+export const WithAnotherIcon = Template.bind({});
+WithAnotherIcon.args = {
+  text: 'Button',
+  icon: < Icon.Eve/>
+};
```
Comment (Wolshebnik) [inline src/components/back-button/BackButton.stories.tsx:24]: add empty string to end

File: src/components/back-button/index.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import {Icon} from '..';
+
+import * as Styles from './styles';
+import { IBackButton } from './types';
+
+const BackButton = ({ onClick, icon, text, ...props }:IBackButton) => {
+  const navigate = useNavigate();
+
+  return (
+
```
Comment (Wolshebnik) [inline src/components/back-button/index.tsx:12]: Delete space

File: src/components/back-button/BackButton.stories.tsx (hunk 1 above)
Comment (Voloshkina) [reply src/components/back-button/BackButton.stories.tsx]: added

File: src/components/back-button/index.tsx (hunk 2 above)
Comment (Voloshkina) [reply src/components/back-button/index.tsx]: done

File: src/components/back-button/BackButton.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import * as Icon from '../icon';
+
```
Comment (Wolshebnik) [inline src/components/back-button/BackButton.stories.tsx:4]: Remove space and sort

File: src/components/back-button/index.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { useNavigate } from 'react-router-dom';
+
+import * as Icon from '../icon';
+
```
Comment (Wolshebnik) [inline src/components/back-button/index.tsx:4]: the same

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**